### ISSUE TITLE
Build.sh fails to recognize libswresample on Fedora

### ISSUE THREAD

**Issue by menaechmi** [clementgallet/libTAS#304]
Greetings!

**Describe the bug:**
On both the most recent release and the git version of libTAS, trying to run build.sh fails on Fedora. config.ac expects these headers files to be at `include/libswresample/`

All ffmpeg headers on Fedora are located in `/usr/include/ffmpeg/lib*`. In a previous version, it was possible to get it to work by editing config.ac to include the extra folder, but with recent changes that doesn't work. Creating a symlink does, though.

**To reproduce:**
1. Use a Fedora-based distro (presumably there are others it would fail on also)
2. Download libTAS and all pre-reqs
3. Run build.sh (either with or without i368)

**Relevant part of conf.log:**
```
configure:5662: checking libswresample/swresample.h usability
configure:5662: gcc -c -g -O2  conftest.c >&5
In file included from conftest.c:69:
/usr/include/libswresample/swresample.h:123:10: fatal error: libavutil/channel_layout.h: No such file or directory
  123 | #include "libavutil/channel_layout.h"
```
**System information:**
Fedora 31 (Kernel 5.4.15-200)
Also Fedora 30 (Kernel 5.3.7-200)

**Comment by clementgallet** [comment]
Mhhh, so the extra include path `/usr/include/ffmpeg/` is required because libswresample includes a file from libavutil? So I guess I should add it using pkgconfig then. Would  `./build.sh CXXFLAGS=-I/usr/include/ffmpeg/` work for you?

**Comment by menaechmi** [comment]
So the libswresample headers are located at `/usr/include/ffmpeg/libswresample/` (which only includes swresample.h and version.h) and the libavutil files are at `/usr/include/ffmpeg/libavutil/` as both are provided by the ffmpeg-libs package.

That CXXFLAGS did not work. Running `ln -s /usr/include/ffmpeg/libswresample /usr/libswresample` and `ln -s /usr/include/ffmpeg/avutil/ /usr/include/avutil` leads to the build script completing, but compilation fails with a no such file or directory error for the following:
GameLoop_moc.cpp
ui/AnnotationsWindow_moc.cpp
ui/AutoSaveWindow_moc.cpp
ui/ControllerAxisWidget_moc.cpp
ui/AutoSaveWindow_moc.cpp

[log_file_stderror.txt](https://github.com/clementgallet/libTAS/files/4149721/log_file_stderror.txt)
[log_file_failedcomp.txt](https://github.com/clementgallet/libTAS/files/4149722/log_file_failedcomp.txt)

**Comment by menaechmi** [comment]
The moc compilation error is caused because the command in Fedora (SE Linux in general?) is `moc-qt5`, not just `moc`. Easily fixed by setting the MOC environment variable when running the build script:
`MOC=moc-qt5 ./build.sh`

**Comment by clementgallet** [comment]
Both issues should be fixed in e53925e5b7cbdb6df0e1393b63dca09cd6415121
